README.md

```diff
@@ -1,2 +1,95 @@
 # gnosis-labs-zuberlin2024
+
 Repository for the hackathon run by Gnosis Labs at ZuBerlin 2024.
```

Comment:
I think it would be nice to have a bit of an intro here (with a summary of what the `Task` is). Something like:

Welcome to the Gnosis AI ZuBerlin 2024 Hackathon repo! Here you will find all you need to build an AI Agent that can make predictions on outcomes of future events.

Follow the instructions below to get started.

Reply:
Good idea, added